Mathport/Syntax/Translate/Basic.lean

```diff
@@ -1282,9 +1282,17 @@ def trCommand' : Command → M Unit
       warn! "unsupported: hide command"
       -- pushM `(hide $(ops.map fun n => mkIdent n.kind)*)
   | Command.theory #[⟨_, Modifier.noncomputable⟩] =>
-    pushM `(command| noncomputable theory)
-  | Command.theory #[⟨_, Modifier.doc doc⟩, ⟨_, Modifier.noncomputable⟩] =>
-    pushM `(command| $(trDocComment doc):docComment noncomputable theory)
+    pushM `(command| noncomputable section)
+  -- FIXME: Lean 4 does not currently allow a doc comment on
+  -- `noncomputable section`.
```

Comment:
@gebner, I will probably merge this PR if CI passes, but I'd be happy to hear your take on this issue.

Reply:
I didn't even know you could put doc comment on `noncomputable theory`.  I think the proper solution here is to add a `warn!` which prints a warning comment in the produced file.